src/modules/pos/validator/components/ValidatorsTable/index.js

```diff
@@ -35,7 +45,7 @@ const ValidatorsTable = ({ setActiveTab, activeTab, blocks, filters }) => {
     [activeTab, sort, filters]
   );
   const { data: validators } = useValidators({
-    config: { params: { limit: 103, status: 'active' } },
+    config: { params: { limit: ROUND_LENGTH, status: 'active' } },
```

Comment:
I'd recommend keeping it as 103, since that's the max limit the API supports. Or, if using the `ROUND_LENGTH`, apply paginated query technique to fetch all the data, since a custom application can possible have a much larger round and this query then will return validation errors from Lisk Service.

```suggestion
    config: { params: { limit: 103, status: 'active' } },
```

Reply:
@sameersubudhi Fixed